models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "1.0.0",
+    "changelog": {
+        "1.0.0": "Initial release"
+    },
+    "monai_version": "1.1.0",
```

Comment:
Hi @Can-Zhao , may need to modify the monai version into the right one, same for pytorch version (other bundles use 1.13.1)

Reply:
Yes, it is updated